.cirrus.yml

```diff
@@ -209,7 +209,7 @@ centos7_task:
 
 debian12_task:
   container:
-    # Debian 12 (bookworm) EOL: (not yet released)
+    # Debian 12 (bookworm) EOL: TBD
     dockerfile: ci/debian-12/Dockerfile
```

Comment:
Seems reasonable to remove SKIP_TASK_ON_PR and add it to debian11 (and 10?)

Reply:
Could also see bumping the arm_debian11 build to debian12 instead.

Reply:
Thanks, done.